src/explorer/send_query.ts

```diff
@@ -0,0 +1,23 @@
+import { request } from 'graphql-request';
```

Comment:
Don't need this file anymore since. you're moving `getBridgeTxnInfo` into `query.ts`